types/escape-regexp/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for escape-regexp 0.0
+// Project: https://github.com/baz/foo (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
```

Comment:
Need project url here.

Reply:
My bad, didn't realize it was not pulled. Original author did not link to git repo, so I only have npmjs link, hope it's enough.